test/unit/sdam/spec.test.js

```diff
@@ -51,11 +51,21 @@ describe('Server Discovery and Monitoring (spec)', function() {
     serverConnect.restore();
   });
 
+  const shouldSkip = desc => {
+    const descriptions = [
+      'Monitoring a standalone connection',
+      'Monitoring a standalone connection - suppress update events for equal server descriptions'
+    ];
```

Comment:
[skipped the tests b/c this convo here](https://mongodb.slack.com/archives/C72LB5RPV/p1588864275286000)

Reply:
Is there a DRIVERS ticket you can link to which fixes the change? Please include it here

Reply:
adding comment to test file

// DRIVERS-1249 should add directConnection and then update spec, remove skip

https://jira.mongodb.org/browse/DRIVERS-1249